src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -659,7 +659,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                     ->beforeNormalization()
                         ->ifTrue(function ($v) { return isset($v['strict_email']); })
                         ->then(function ($v) {
-                            @trigger_error('The "framework.validation.strict_email" configuration key has been deprecated in Symfony 4.1. Use the "framework.validation.email_validation_mode" configuration key instead.', E_USER_DEPRECATED);
+                            @trigger_error('The "framework.validation.strict_email" configuration key is deprecated since Symfony 4.1. Use the "framework.validation.email_validation_mode" configuration key instead.', E_USER_DEPRECATED);
```

Comment:
Unrelated changes should really be moved to another PR. This is especially important here as people will have a look at this PR when upgrading.